stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.0_10_0.mlir

```diff
@@ -1,10 +1,11 @@
 // RUN: stablehlo-opt --mlir-print-op-generic %s.bc | FileCheck %s
 // RUN: stablehlo-translate --deserialize %s.bc | stablehlo-translate --serialize --target=0.10.0 | stablehlo-opt --mlir-print-op-generic | FileCheck %s
-// RUN: stablehlo-translate --deserialize %s.bc | stablehlo-opt > %t.0
-// RUN: stablehlo-opt --strip-debuginfo %s > %t.1
-// RUN: diff %t.0 %t.1
-// RUN: stablehlo-translate --serialize --target=0.10.0 --strip-debuginfo %s > %t.2
-// RUN: diff %s.bc %t.2
+// RUN: diff <(stablehlo-translate --deserialize %s.bc | stablehlo-opt) <(stablehlo-opt --strip-debuginfo %s)
```

Comment:
https://github.com/openxla/stablehlo/pull/1764

Can we avoid this refactoring?

Reply:
Oh, good to know! Yeah I changed it back to use temp files.